templates/apps/sumo-aws-apps/firewall-manager-org/firewall-manager-org-waf-network-firewall-policy.yaml

```diff
@@ -316,9 +310,7 @@ Resources:
                     - KinesisFirehoseDeliveryStreamArn: !If [create_delivery_stream, !GetAtt KinesisFirehoseDeliveryStream.Arn ,''] 
                 - ''
  
-  rFmsRegionalWafLinuxPolicy:
-    DependsOn:
```

Comment:
these policies needs KinesisFirehoseDeliveryStreamArn so it should exists before we create these policies

Reply:
if Create Delivery Stream then the LoggingConfiguration will need KinesisFirehoseDeliveryStreamArn. In case Create Delivery Stream is not used, LoggingConfiguration will not be configured